projects/architecture/interfaces/primitives/transactions.proto

```diff
@@ -34,3 +27,13 @@ message SubscriptionSignedTransaction {
   Transaction transaction = 1;
   TransactionSubscriptionAppendix transactionSubscriptionAppendix = 2;
 }
+
+message TransactionReceipt {
+  bytes txHash = 1;
+  bool success = 2; // TODO: This is just a temporary field to distinguish between successful and failed transactions
```

Comment:
Why not `string status`?

Reply:
This transaction receipt is very basic.  We'll extend it once @OdedWx will introduce the new structure